app/views/shared/_tasklist_sidebar.html.erb

```diff
@@ -0,0 +1,6 @@
+<aside class="qa-tasklist-sidebar">
+  <h2 class="task-list-title">
+    <a href="<%= tasklist[:base_path] %>"><%= tasklist[:title] %></a>
+  </h2>
+  <%= render "govuk_component/task_list", tasklist[:tasklist] %>
```

Comment:
I think you missing the `small: true` attribute here which is used for the sidebar.

Reply:
It's in the json config (which is therefore slightly different from the collections one) Also, heading_level: 3